### ISSUE TITLE
:'( what happened to the mypal github 

### ISSUE THREAD

**Issue by Leifman27** [Feodor2/Mypal2#2]

**Comment by chermany4ever** [comment]
Thanks feodor, man yours was a truly great job. I was reading all the crap but they deleted the whole thing. Very sad man. Very sad. Many silenced are with you.

**Comment by Leifman27** [comment]
Yeah i am all team Feodor. I hope he doesn't stop. even off github or whatever.... hell i'd even pay for new builds.

**Comment by Feodor2** [comment]
I have transfered the issue here.
